designer/server/src/main/scala/pl/touk/nussknacker/ui/api/NodesResources.scala

```diff
@@ -45,13 +55,13 @@ import scala.concurrent.{ExecutionContext, Future}
   */
 class NodesResources(val processRepository: FetchingProcessRepository[Future],
                      subprocessRepository: SubprocessRepository,
-                     typeToConfig: ProcessingTypeDataProvider[ModelData],
+                     typeToConfigModel: ProcessingTypeDataProvider[ModelData],
+                     typeToConfigPrcessing: ProcessingTypeDataProvider[ProcessingTypeData],
```

Comment:
You can avoid adding `typeToConfigPrcessing`, use `val modelData = typeToConfig.forTypeUnsafe(process.processingType)` in line 68